contracts/Insurance.sol

```diff
@@ -124,7 +130,12 @@ contract Insurance is IInsurance, Ownable, SafetyWithdraw {
         int256 quote = (tracer.getBalance(address(this))).position.quote;
         if (quote > 0) {
             tracer.withdraw(uint256(quote));
-            collateralAmount = collateralAmount + uint256(quote);
+            if (collateralAmount > 0) {
+                collateralAmount = collateralAmount + uint256(quote);
+            } else {
+                // Pay to buffer if nothing in public insurance
+                bufferAmount = bufferAmount + uint256(quote);
+            }
```

Comment:
Some of this is true, some is incorrect. 

Insurance funding rate payments are paid proportionally to the two pools. e.g. If insurance funding payment = 25 Quote, Insurance buffer account has 10 units of quote already in the pool (10% of total), public account has 90 units of quote (90% of total), Insurance buffer account increases by 25*0.1, and public account increases by 25*0.9. 

BUT, if neither pool has any quote collateral in their account (like maybe at genesis), it will go 100% to insurance buffer account.

Reply:
Yep, that makes sense; will fix this up

Reply:
Why do you say 250.1 and 250.9? Where have these numbers come from?

Reply:
Yep, that makes sense; can fix

Reply:
According to @kumar-ish , the buffer account only gets paid when
a) The public account is empty
b) People pay withdrawal fees

But you're saying it's proportional to pool holdings.

If your statement is correct, should this split be done as soon as insurance funding rate payments are made, or is it OK to make this calculation when `updatePoolAmount(...)` is called?

Reply:
This should be fixed now